app/helpers/application_helper.rb

```diff
@@ -66,7 +66,7 @@ def title(page_title)
   end
 
   def ga_scroll_ids(ids)
-    content_for(:ga_scroll_ids) { ids.join(',') }
+    content_for(:ga_scroll_ids) { ids.join(',')+',' }
```

Comment:
I'm still unsure about the behaviour and scope here, possibly because of the naming. `ids.join()` will take`ids` param and turn it from an array obj into a string, and provide it back out on `content_for` in another template. Is it storing ids implicitly in the method scope? how does it not get overridden in a second call with an embedded partial template?